src/components/views/messages/MLocationBody.tsx

```diff
@@ -84,8 +97,14 @@ export default class MLocationBody extends React.Component<IBodyProps> {
     };
 
     render() {
+        const error = this.state.fail ?
+            <div className="mx_EventTile_tileError mx_EventTile_body">
+                Failed to load map - check config.json has a valid API key
```

Comment:
needs i18n

Reply:
also probably not a great user-facing error given majority of users won't be in control of the config.json, this should probably go into the console with a glossier user-facing error